src/elements/select/select.ts

```diff
@@ -71,17 +85,11 @@ export class SbbSelectElement extends SbbUpdateSchedulerMixin(
     didClose: 'didClose',
   } as const;
 
-  /** The value of the select component. If `multiple` is true, it's an array. */
-  @property() public value?: string | string[];
```

Comment:
Open question about the value in select; Should we refactor this along the tag-group, where on client side, the value is implemented as a getter/setter, which just checks for selected options?
What do you think?

Reply:
In general, I agree with you (the `radio-button-group` will be like that) but I feel the `sbb-select` is a bit too complex for that. Too many references to `value` and the initialization is what concerns me the most

Reply:
Fair enough. Maybe we can revisit this in the future, but it is fine for me for now. 👍